docs/source/install.rst

```diff
@@ -35,12 +35,32 @@ Requirements
 ------------
 
 Todoman requires python 3.3 or later. Installation of required libraries can be
-done via pip, or your OS's package manager. If you're interested in packaging
-todoman, all dependencies are listed in requirements.txt_.
+done via pip, or your OS's package manager.
 
 Todoman will not work with python 2. However, keep in mind that python 2 and
 python 3 can coexist (and most distributions actually ship both).
 
 Recent versions also have experimental support for pypy3.
 
+.. _notes-for-packagers:
+
+Notes for Packagers
+-------------------
+
+All of todoman's dependencies are listed in the requirements.txt_ file. New
+dependencies will be clearly announced in the ``CHANGELOG.rst`` file for each
+release. Patch releases (eg: those where the third digit of the version is
+incremented) **will not** introduce new dependencies.
+
+If your packages are generated by running ``setup.py install`` or some similar
```

Comment:
Yup, also affects `setup.py install`. The AUR package currently uses this and takes ~1s extra comparted to this script.